### PR TITLE
LPS-144314 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-144314

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Just start reviewing

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 1]
```diff
@@ -513,6 +513,21 @@ definition {
 		PortletEntry.save();
 	}
 
+	macro deleteFriendlyURL {
+		Navigator.gotoNavTab(navTab = "General");
+
+		PortletEntry.changeLocale(
+			fieldName = "Name",
+			locale = "${localizedFriendlyURL}");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:522]: locale = "${locale}"

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 2]
```diff
@@ -733,6 +789,50 @@ definition {
...
+				locale = "${friendlyUrlLocalized}");
+		}
+
+		AssertTextEquals(
+			inputFieldId = "friendlyURL",
+			locator1 = "TextInput#INPUT_ID",
+			value1 = "/${pageName}");
+	}
+
+	macro viewNoOldFriendlyURL {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:901]: viewOldFriendlyURLNotPresent

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 3]
```diff
@@ -818,6 +818,33 @@ definition {
 		}
 	}
 
+	macro uploadFileForLogo {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:821]: uploadLogo

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 4]
```diff
@@ -818,6 +818,33 @@ definition {
...
+
+		UploadCommonFile.uploadCommonFileHiddenNoMouseOver(
+			locator1 = "TextInput#FILE",
+			value1 = "${logoFile}");
+
+		Pause(locator1 = "10000");
+
+		Button.clickDone();
+
+		Pause(locator1 = "5000");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:841]: Remove this line

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 5]
```diff
@@ -950,6 +950,12 @@ definition {
 			locator1 = "PagesAdmin#LIST_GROUP_ITEM");
 	}
 
+	macro viewPageLayoutColumn {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:953]: Duplicate with Page.viewSelectedLayoutColumn

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you update according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you update according to comments? Thanks

Hi @Tim-Cao, I updated the PR with some additional commits according your comments. Could you take a look? Thanks!

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**